ios/RNMagicScript/components/UiNodes/UiAudioNode.swift

```diff
@@ -54,16 +54,16 @@ import SceneKit
     @objc var streamedFileOffset: CGFloat = 0 {
         didSet { setNeedsLayout() }
     }
-    var spatialSoundPosition: SpatialSoundPosition? {
-        didSet { soundNode.position = spatialSoundPosition?.position ?? SCNVector3Zero }
+    var spatialSoundPosition: [SpatialSoundPosition] = [] {
+        didSet { soundNode.position = spatialSoundPosition.first?.position ?? SCNVector3Zero }
     }
-    var spatialSoundDirection: SpatialSoundDirection? {
-        didSet { soundNode.direction = spatialSoundDirection?.direction ?? SCNQuaternionIdentity }
+    var spatialSoundDirection: [SpatialSoundDirection] = [] {
+        didSet { soundNode.direction = spatialSoundDirection.first?.direction ?? SCNQuaternionIdentity }
     }
-    //var spatialSoundDistance: SpatialSoundDistance?
-    //var spatialSoundRadiation: SpatialSoundRadiation?
-    //var spatialSoundDirectSendLevels: SpatialSoundSendLevels?
-    //var spatialSoundRoomSendLevels: SpatialSoundSendLevels?
+    //var spatialSoundDistance: [SpatialSoundDistance]
+    //var spatialSoundRadiation: [SpatialSoundRadiation]
+    //var spatialSoundDirectSendLevels: [SpatialSoundSendLevels]
+    //var spatialSoundRoomSendLevels: [SpatialSoundSendLevels]
```

Comment:
remove these comments

Reply:
These are missing properties that should be implemented in future.